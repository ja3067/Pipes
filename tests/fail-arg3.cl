if True:
    def foo(x : int):
        return x

print(foo(3))
print(foo(3.0))
