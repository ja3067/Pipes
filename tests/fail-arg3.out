3
RuntimeError: invalid type assigned to x (expected int)
